### DISCUSSION TITLE
TASK [k3s_server : Verify that all nodes actually joined (check k3s-init.service if this fails)] *** Keeps failing

### DISCUSSION THREAD

**Discussion by Zeroxcharisma** [techno-tim/k3s-ansible#439] [Questions]
![image](https://github.com/techno-tim/k3s-ansible/assets/79290030/1d84b1be-f1e2-4f33-bf84-25124a7d1b43)
I don't know what I am missing here, do you happen to have any solutions? 
where should I start checking?

**Comment by pbolduc** [comment]
Go to your master node and check if k3s is running (use `journalctl`). If the /boot/cmdline.txt or /boot/firmware/cmdline.txt is not properly configured with cgroups, k3s will be in a crash loop. There was a PR merge today to fix the issue with Debian Bookwork where in Bookwork, the cmdline.txt file was moved to /boot/firmware/cmdline.txt.

https://github.com/techno-tim/k3s-ansible/pull/456
